Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -685,25 +632,21 @@ private void chooseController() {
 		AlertDialog.Builder builder = new AlertDialog.Builder(this);
 		builder.setTitle(R.string.emulation_choose_controller);
 		builder.setSingleChoiceItems(R.array.controllersEntries, mPreferences.getInt("wiiController", 3),
-				new DialogInterface.OnClickListener() {
-					@Override
-					public void onClick(DialogInterface dialog, int indexSelected) {
-						editor.putInt("wiiController", indexSelected);
-
-						NativeLibrary.SetConfig("WiimoteNew.ini", "Wiimote1", "Extension",
-								getResources().getStringArray(R.array.controllersValues)[indexSelected]);
-					}
-				});
-		builder.setPositiveButton(getString(R.string.ok), new DialogInterface.OnClickListener() {
-			@Override
-			public void onClick(DialogInterface dialogInterface, int i) {
-				editor.apply();
-
-				mEmulationFragment.refreshInputOverlay();
-
-				Toast.makeText(getApplication(), R.string.emulation_controller_changed, Toast.LENGTH_SHORT).show();
-			}
-		});
+				(dialog, indexSelected) ->
+				{
+                    editor.putInt("wiiController", indexSelected);
+
+                    NativeLibrary.SetConfig("WiimoteNew.ini", "Wiimote1", "Extension",
+                            getResources().getStringArray(R.array.controllersValues)[indexSelected]);
+                });
+		builder.setPositiveButton(getString(R.string.ok), (dialogInterface, i) ->
+		{
+            editor.apply();
+
+            mEmulationFragment.refreshInputOverlay();
+
+            Toast.makeText(getApplication(), R.string.emulation_controller_changed, Toast.LENGTH_SHORT).show();
+        });
```

Comment:
The ending curly brace should be aligned with the starting curly brace (here and 8 lines up).